Automattic/jetpack
# Social: Fix the check for Social Image Generator for the media picker

The check for Social Image Generator was checking the current post config twice, rather than the config for the post and the feature availability, and so the media picker was never being disabled.





## Proposed changes:

This swaps the `isSocialImageGeneratorEnabled` prop for `isSocialImageGeneratorAvailable` so we check for the feature availability

### Other information:


## Does this pull request change what data or activity we track or use?


No


## Testing instructions:






* With a site with the advanced plan, enable SIG in the sidebar
* With this change the media picker should be disabled with a notice. Without it, nothing will happen with the media picker.
